beacon_node/lighthouse_network/src/peer_manager/mod.rs

```diff
@@ -61,6 +61,8 @@ pub const MIN_OUTBOUND_ONLY_FACTOR: f32 = 0.2;
 /// limit is 55, and we are at 55 peers, the following parameter provisions a few more slots of
 /// dialing priority peers we need for validator duties.
 pub const PRIORITY_PEER_EXCESS: f32 = 0.2;
+/// The number of inbound peers that are connected that indicate this node is not behind a NAT.
+pub const INBOUND_PEERS_NAT: usize = 5;
```

Comment:
Why 5? How's the node getting any inbound peers at all if it does not have a reachable address?!

Reply:
I think the node may still be reached by peers that are public, i.e. the node dials a public peer so the router adds that public address to it's table and therefore it may be dialed from them in the future right?